app/controllers/v1/events_controller.rb

```diff
@@ -9,7 +9,7 @@ def create_rsvp
     person = create_person_and_action(default_template_id: Activity::DEFAULT_TEMPLATE_IDS[:rsvp])
 
     response, status = rescue_error_messages do |variable|
-      Integration::NationBuilder.create_person_and_rsvp(event_id: params[:event_id], person_attributes: person.attributes)
+      Integration::NationBuilder.create_person_and_rsvp(event_id: params[:event_id], person_attributes: person.attributes.symbolize_keys)
```

Comment:
the test for this was broken. In trying to fix the test I found that person attributes were being sent to Integration::NationBuilder incorrectly. In future this should make use of PersonWithRemoteFields and ActiveJob